### PR TITLE
Adding my simple demo to the Demos folder

### PR DESCRIPTION
**Pull request by shoemakerc**

### CODE REVIEW

**Review by cpimhoff — APPROVED** [review]
Woot!

**Comment by brennan724** [comment]
We should approve this, guys!

File: Demos/shoemakerc/Assets/Characters/FirstPersonCharacter/Prefabs/FPSController.prefab [hunk 1]
```diff
@@ -0,0 +1,324 @@
+%YAML 1.1
```
Comment (grahamearley) [inline Demos/shoemakerc/Assets/Characters/FirstPersonCharacter/Prefabs/FPSController.prefab:1]: Comments!

**Review by Donahuec — APPROVED** [review]
Yay~!!

File: Demos/shoemakerc/Assets/Characters/FirstPersonCharacter/Scripts/FirstPersonController.cs [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+            m_NextStep = m_StepCycle/2f;
+            m_Jumping = false;
+            m_AudioSource = GetComponent<AudioSource>();
+			m_MouseLook.Init(transform , m_Camera.transform);
+        }
+
+
+        // Update is called once per frame
+        private void Update()
+        {
```
Comment (grahamearley) [inline Demos/shoemakerc/Assets/Characters/FirstPersonCharacter/Scripts/FirstPersonController.cs:63]: Another comment. This is neat

**Review by grahamearley — APPROVED** [review]
Overall, a great PR. I approve.

**Review by mjavaly — APPROVED** [review]
nice!

**Review by brennan724 — APPROVED** [review]
looks good

**Pull request merged**